qt/python/mantidqt/widgets/sliceviewer/test/test_sliceviewer_presenter.py

```diff
@@ -16,11 +16,12 @@
 # Mock out simpleapi to import expensive import of something we don't use anyway
 sys.modules['mantid.simpleapi'] = mock.MagicMock()  # noqa: E402
 
-from mantidqt.widgets.sliceviewer.model import SliceViewerModel, WS_TYPE
-from mantidqt.widgets.sliceviewer.presenter import (PeaksViewerCollectionPresenter, SliceViewer)
-from mantidqt.widgets.sliceviewer.transform import NonOrthogonalTransform
-from mantidqt.widgets.sliceviewer.toolbar import ToolItemText
-from mantidqt.widgets.sliceviewer.view import SliceViewerView, SliceViewerDataView
+from mantidqt.widgets.sliceviewer.model import SliceViewerModel, WS_TYPE  # noqa: E402
+from mantidqt.widgets.sliceviewer.presenter import (  # noqa: E402
+    PeaksViewerCollectionPresenter, SliceViewer)
+from mantidqt.widgets.sliceviewer.transform import NonOrthogonalTransform  # noqa: E402
+from mantidqt.widgets.sliceviewer.toolbar import ToolItemText  # noqa: E402
+from mantidqt.widgets.sliceviewer.view import SliceViewerView, SliceViewerDataView  # noqa: E402
```

Comment:
More out of curiosity, why do these now need noqa tags?

Reply:
My local flake8 required them I think because of line 17 being a non import line and then there being more imports afterwards. Actually I think line 17 itself doesn't need it...